immunisations/README.md

```diff
@@ -0,0 +1,149 @@
+# Immunization Mapping
+
+## JSON FHIR > XML HL7
+
+An Immunization is mapped to a ObservationStatement.
+
+| Mapped to (JSON FHIR Immunization field)                 | Mapped from (XML HL7 / other source)                                                                          |
+|----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| id                                                       | `ObservationStatement / id / root]`                                                                           |
+| meta / profile                                           | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Immunization-1"`                 |
+| extension / url                                          | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-VaccinationProcedure-1` |

```

Comment:
I think we should use `extension\[0]` and `extension\[1]` to differentiate the two different extensions being added.